client/lib/pages/main_pages/recent_numbers.dart

```diff
@@ -179,9 +179,9 @@ class _RecentNumbersPageState extends State<RecentNumbersPage> {
   Map<DataAggregation, Widget> _buildSegmentControlChildren(
       BuildContext context, DataAggregation selectedValue) {
     var valueToDisplayText = {
+      DataAggregation.daily: 'Daily',
       DataAggregation.total: S.of(context).latestNumbersPageTotalToggle,
       // TODO: This string changed, so we must change regenerate translation keys someday.
```

Comment:
Remove this line, and add it up above the text **Daily** on line 182
```suggestion
```

Reply:
@theswerd Made the recommended change!

Reply:
Halfway done - you need to add it back above the unlocalized text